tests/xapi-plugins/test_raid.py

```diff
@@ -0,0 +1,2 @@
+def test_check_raid_pool(host):
+    host.call_plugin('raid.py', 'check_raid_pool')
```

Comment:
Doesn't this raise when there's no RAID, currently?

Reply:
Yes, i'll add a fixture that creates the RAID if it doesn't exist and that'll remove it if it has been created for the tests.

Reply:
Do I wait for this or do I approve now?

Reply:
please wait, i'm working on it, I'd like to be sure no test raise unexpectidely before we merge. :)

Reply:
You can approve now, after checking my fixture :)